modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -358,10 +363,21 @@ To view the latest processed transaction IDs (and other metrics) in Neo4j Browse
 ====
 
 ==== Targeting multiple servers
+
 It is recommended to provide a list of multiple target servers when taking a backup from a cluster, since that may allow a backup to succeed even if some server is down, or not all databases are hosted on the same servers.
 If the command finds one or more servers that do not respond, it continues trying to backup from other servers and continues backing up other requested databases, but the exit code of the command is non-zero, to alert the user to the fact there is a problem.
 If a name pattern is used for the database together with multiple target servers, all servers contribute to the list of matching databases.
 
+[role=label--new-2025.09]
+==== Using `--remote-address-resolution`
+
+Starting from 2025.09, the `--remote-address-resolution` option is available.
+When enabled, the DBMS automatically selects the most appropriate servers to act as backup sources for a given database.
```

Comment:
@jackwaudby hello! Thanks for approving the PR, but I'm wondering if we need to clarify what 'appropriate servers' mean. Available servers? Or the most up to date? What will happen if there are some 'dead' servers in the cluster that are not cordoned? Will it break the command?